site/content/en/docs/Installation/upgrading.md

```diff
@@ -51,6 +51,74 @@ The following are steps to implement this:
+and the feature is currently in `Dev`. Please continue to use the multi-cluster strategy for
+production critical upgrades. Feedback on the upcoming `Alpha` and `Beta` releases is appreciated.
+{{< /alert >}}
+
+For In-Place Agones Upgrades we highly recommend installing using Helm. Helm has a significant
+advantage over `install.yaml` in that Helm automatically rolls back the upgrade if the agones-system
+pods are not all successfully running within the designated `--timeout`.
+
+Note: By “controller derived configuration” we mean the parts of the Game Server’s final state that
+are not part of the Game Server spec template that are instead passed to the Game Server through the
+controller, such as the FEATURE_GATES or the Agones SDK Image.
+
+##### Upgrade Existing Fleet (recommended)
+
+1. Run `helm upgrade my-release --install --atomic --wait --timeout 10m --namespace=agones-system`
+with all the appropriate arguments, such a `--version`, for your specific upgrade. Keep in mind that
+`helm upgrade` overwrites all `--set agones.` arguments, so these must be set for each upgrade. See
+[Helm Configuration]({{< relref "./Install Agones/helm.md" >}}) for a list of all available
+configurable parameters.
+2. Wait until the `helm upgrade` is complete.
+3. Kick off rolling update of existing Fleet.
+    1. Make any change to the Fleet Spec Template in your fleet.yaml and reapply with
+    `kubectl apply -f fleet.yaml`.
+    2. If you have no changes to make to the existing Fleet Spec Template in the fleet.yaml,
+    then add a label to the spec.template.metadata.labels. This has the added advantage that Game
+    Servers can be selected for allocation by label. Reapply with `kubectl apply -f fleet.yaml`.
+    ```yaml
+    apiVersion: agones.dev/v1
+    kind: Fleet
+    metadata:
+      name: example-fleet
+    spec:
+      replicas: 2
+      template:
+        metadata:
+          # Adding a label will start a Fleet rollout (RollingUpdate or Recreate based on the replacement strategy type) with the most up to date Agones config. The label can be any `key: value` pair.
+          labels:
+            release: 1.42.0
+    ```
+
+    3. This creates a new Game Server Set. Ready Game Servers will shut down and be recreated on
+    the new configuraiton on the new Game Server Set. Allocated Game Servers will use the existing
+    configuration, and once they are set back to Ready, these Game Servers will shut down and be
+    replaced by Ready Game Servers at the new configuration on the new Game Server Set.
+    4. The rollout is complete once there is one Game Server Set at the new configuration.
+4. Run any other tests to ensure the Agones installation is working as expected.
+5. Congratulations - you have now upgraded to a new version of Agones! 👍
+
+##### Delay Upgrade to Existing Fleet (not recommended)
+
+1. Run `helm upgrade my-release --install --atomic --wait --timeout 10m --namespace=agones-system`
```

Comment:
Rather than have this section, could we not have 

#### In-Place Agones Upgrades

... outline how to do it in in place

##### Upgrade to Existing Fleet (recommended)

... outline how to force a `Fleet` update

and then we don't have to have the same thing twice?